Jan2016/CJ/com/two95/college/Courses.java

```diff
@@ -0,0 +1,64 @@
+package com.two95.college;
+
+import java.util.Collections;
+import java.util.List;
+
+public class Courses {

```

Comment:
Shouldn't the class name be Course? Courses will be a more than one course..usually used in a list etc